src/jit/lsra.cpp

```diff
@@ -196,7 +196,9 @@ unsigned LinearScan::getWeight(RefPosition* refPos)
             // the basic block weight in which they appear.
             // However, it is generally more harmful to spill tree temps, so we
             // double that.
-            weight = 4 * blockInfo[refPos->bbNum].weight;
+            const unsigned TREE_TEMP_REF_COUNT    = 2;
+            const unsigned TREE_TEMP_BOOST_FACTOR = 2;
```

Comment:
Eventually we could migrate these heuristic constants into a static class called HeuristicConstants